common/pb/session_commands.proto

```diff
@@ -160,3 +163,30 @@ message Command_AccountPassword {
     optional string old_password = 1;
     optional string new_password = 2;
 }
+
+message Command_ForgotPasswordRequest {
+    extend SessionCommand {
+        optional Command_ForgotPasswordRequest ext = 1021;
+    }
+    optional string user_name = 1;
+    optional string clientid = 2;
+}
+
+message Command_ForgotPasswordReset {
+    extend SessionCommand {
+        optional Command_ForgotPasswordReset ext = 1022;
+    }
+    optional string user_name = 1;
```

Comment:
I can make any of them required that you like, I didnt want to cause any issue's with future protocol manipulations on this process that's all.  I'll make user_name for sure.